graylog2-server/src/main/java/org/graylog2/inputs/codecs/JsonPathCodec.java

```diff
@@ -84,6 +85,9 @@ public Message decode(@Nonnull RawMessage rawMessage) {
                 fields = flatten(json);
             } catch (JsonFlattenException e) {
                 LOG.warn("JSON contains type not supported by flatten method.", e);
+            } catch (JsonProcessingException e) {
+                LOG.warn("JsonProcessingException at JsonPathCodec.decode().", e);
```

Comment:
This is just a repetition of the exception, which will show up in the server log. A better error message could be:
```java
LOG.warn("Error while parsing JSON of message: ", e);
```